applications/plugins/SensableEmulation/CMakeLists.txt

```diff
@@ -18,13 +18,13 @@ find_package(Sofa.Config REQUIRED)
 sofa_find_package(SofaHaptics REQUIRED)
 sofa_find_package(Sofa.Component.Controller REQUIRED)
 sofa_find_package(Sofa.GL.Component.Rendering3D REQUIRED)
-sofa_find_package(Boost QUIET COMPONENTS thread REQUIRED)
+sofa_find_package(Boost COMPONENTS thread REQUIRED)
 
 include_directories(${Boost_INCLUDE_DIRS})
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES} ${README_FILES})
 set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-DSOFA_BUILD_SENSABLEEMULATIONPLUGIN")
-target_link_libraries(${PROJECT_NAME} SofaHaptics Sofa.Component.Controller Sofa.GL.Component.Rendering3D ${Boost_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} SofaHaptics Sofa.Component.Controller Sofa.GL.Component.Rendering3D Boost::boost)
```

Comment:
when you add `Boost::boost`, it will choose the libboost-thread.so/lib/dylib automatically ??

Reply:
Nah, I should have used boost::thread :face_with_head_bandage: